Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/LoadTestDataPatientBean.java

```diff
@@ -83,82 +113,246 @@ public void setSelectedPatient(Patient selectedPatient) {
         this.selectedPatient = selectedPatient;
     }
 
-    // DATABASE-METHODS-getAll, delete, save
+    // selected-record
+    public Personname getSelectedPersonname() {
+        return selectedPersonname;
+    }
+
+    public void setSelectedPersonname(Personname selectedPersonname) {
+        this.selectedPersonname = selectedPersonname;
+    }
+
+    public Identifier getSelectedIdentifier() {
+        return selectedIdentifier;
+    }
+
+    public void setSelectedIdentifier(Identifier selectedIdentifier) {
+        this.selectedIdentifier = selectedIdentifier;
+    }
+
+    public Address getSelectedAddress() {
+        return selectedAddress;
+    }
+
+    public void setSelectedAddress(Address selectedAddress) {
+        this.selectedAddress = selectedAddress;
+    }
+
+    public Phonenumber getSelectedPhonenumber() {
+        return selectedPhonenumber;
+    }
+
+    public void setSelectedPhonenumber(Phonenumber selectedPhonenumber) {
+        this.selectedPhonenumber = selectedPhonenumber;
+    }
+
+    // database-methods
+    public List<Phonenumber> getPhonenumbers() {
+        if (isValidPatientId()) {
+            return loadTestDataService.getAllPhonenumbersBy(getPatientId());
```

Comment:
A method should have only one return statement instead of two. Use a local variable to get the result and return the local variable.